### PR TITLE
src, cargo: Completely remove WebRTCSink.

### PR DESCRIPTION
**Pull request by joaoantoniocardoso**
Removing WebRTC support until add our own implementation using WebRTCBin

### CODE REVIEW

File: src/server/manager.rs [hunk 1]
```diff
@@ -20,31 +18,18 @@ fn json_error_handler(error: JsonPayloadError, _: &HttpRequest) -> actix_web::Er
...
-                info!("Using the www-path: {path:?}.");
-                app = app.service(
-                    actix_files::Files::new("webrtc", format!("{path}/webrtc/"))
-                        .index_file("index.html"),
-                );
-            }
+            // Load static files from configured www-path
+            info!("Using the www-path: {path:?}.");
+            app = app.service(
+                actix_files::Files::new("webrtc", format!("{path}/webrtc/"))
```
Comment (patrickelectric) [inline src/server/manager.rs:30]: Why is this necessary ?

Comment (joaoantoniocardoso) [reply src/server/manager.rs:30]: It's not necessary indeed

File: src/stream/mod.rs [hunk 2]
```diff
@@ -6,5 +6,4 @@ pub mod types;
 pub mod video_stream_redirect;
 pub mod video_stream_rtsp;
 pub mod video_stream_udp;
-pub mod video_stream_webrtc;
 pub mod webrtc;
```
Comment (patrickelectric) [inline src/stream/mod.rs:9]: why this was not removed ?

**Review by patrickelectric — CHANGES_REQUESTED** [review]
It appears that multiple parts of the webrtc are still in the codebase with this PR.
That also includes turn and tokio, anyhow and util

File: src/stream/mod.rs (hunk 2 above)
Comment (patrickelectric) [reply src/stream/mod.rs:9]: And about anyhow ?

Comment (joaoantoniocardoso) [reply src/stream/mod.rs:9]: I left anyhow because I am using it in our new WebRTC implementation, but I can remove and add again later if it makes more sense

Comment (patrickelectric) [reply src/stream/mod.rs:9]: Ok, it's fine since it was explained. But usually removing and adding back is the cleaner solution, since we can't predict future code. 
We should also add a linter to notify about unused includes.

**Review by patrickelectric — APPROVED** [review]

**Comment by rafaellehmkuhl** [comment]
Just to make sure, one using cockpit+webrtc shouldn't update from this point, till we merge the new implementation (in both camera manager and cockpit), right?

**Comment by patrickelectric** [comment]
We're not going to update the camera manager on BlueOS before finishing the new webrtc implementation
